pmd-java/src/main/java/net/sourceforge/pmd/lang/java/ast/ASTExpression.java

```diff
@@ -20,4 +20,30 @@ public ASTExpression(JavaParser p, int id) {
     public Object jjtAccept(JavaParserVisitor visitor, Object data) {
         return visitor.visit(this, data);
     }
+
+    public boolean isStandAlonePrimitive() {
```

Comment:
It was supposed to recognize primitives, the name is not missleading, the implementation is faulty.